index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/index.js" type="module" async></script>
+    <title>Creative Commons Global Summit 2015</title>
+  </head>
+  <body>
```

Comment:
Well done starting your capstone project, you are almost there! However, there are a few things missing that need to be addressed:

Template:
![Captura de pantalla 2022-04-14 a la(s) 7 03 21 p  m](https://user-images.githubusercontent.com/83717285/163496928-9be4cd27-4e2f-496b-aced-6730516e951f.png)

Your design:
![screencapture-127-0-0-1-5500-index-html-2022-04-14-19_03_01](https://user-images.githubusercontent.com/83717285/163496962-fd790f69-8a12-4f4f-b26b-320ed59140d2.png)

- [ ] There's a blank space on the right, it would be better to remove it as this is not part of the template.

Template:

![Captura de pantalla 2022-04-14 a la(s) 7 33 38 p  m](https://user-images.githubusercontent.com/83717285/163497451-72a154bd-e032-467b-b354-2da6275199be.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 7 42 19 p  m](https://user-images.githubusercontent.com/83717285/163498557-4f4ab953-1606-45f4-86f1-91b2a60563f9.png)

- [ ] The overall design of the top section is looking fairly different from the template, let's increase the size of the texts according to the design and reduce the paragraph container.

Template:
![Captura de pantalla 2022-04-14 a la(s) 7 40 08 p  m](https://user-images.githubusercontent.com/83717285/163497998-0cf32c3e-a7f5-441c-8664-ad1a5c574b77.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 7 41 28 p  m](https://user-images.githubusercontent.com/83717285/163499409-5ac44496-a486-4c01-a505-f17ec10494cd.png)

- [ ]  The desktop version of the main program is missing, let's create it according to the template.

Template:
![Captura de pantalla 2022-04-14 a la(s) 7 48 11 p  m](https://user-images.githubusercontent.com/83717285/163499940-8dd4563e-cb3a-4198-a1b7-030f4b15b93b.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 7 47 43 p  m](https://user-images.githubusercontent.com/83717285/163499994-c810d13d-6ec2-4b0f-813c-89015e4aadde.png)

- [ ] The speakers' section is not looking like the template, please add all the missing design elements like in the template.

Template:
![Captura de pantalla 2022-04-14 a la(s) 7 53 02 p  m](https://user-images.githubusercontent.com/83717285/163500881-515b1f8f-cecf-419b-98d1-28122d915179.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 7 50 51 p  m](https://user-images.githubusercontent.com/83717285/163500922-69c4214d-ad10-4a90-8c8e-e71be3a3aa71.png)

- [ ] The partners' section has the logos, well done! But the design is not looking like the template, please add the background and align the logos according to the template.

Template:
![Captura de pantalla 2022-04-14 a la(s) 7 57 59 p  m](https://user-images.githubusercontent.com/83717285/163501242-39319de4-9185-4864-8dac-f5f3ae6192c6.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 7 57 34 p  m](https://user-images.githubusercontent.com/83717285/163501273-4cae81cb-ddf5-4a43-a338-a17213f60ab8.png)

- [ ]  Let's create the footer section, we need to add all the missing design elements.